tools/install_macos_dependencies.sh

```diff
@@ -28,8 +28,11 @@ echo "Downloading libtensorflow..."
 curl https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-darwin-x86_64-1.8.0.tar.gz > libtensorflow.tar.gz
 
 echo "Extracting and copying libtensorflow..."
-sudo tar zxf libtensorflow.tar.gz -C /usr/local
+tar zxf libtensorflow.tar.gz -C .
```

Comment:
It is dangerous to presume you can write into the current directory. You can create a temp  directory in /temp with  mktemp -d.